src/libra_py/dynamics/tsh/compute.py

```diff
@@ -566,7 +566,7 @@ def run_dynamics(dyn_var, _dyn_params, ham, compute_model, _model_params, rnd):
                              "collapse_option":0,  
                              "decoherence_rates":MATRIX(nstates, nstates),
                              "ave_gaps":MATRIX(nstates,nstates),
-                             "schwartz_decoherence_inv_alpha": MATRIX(nstates, 1),
+                             "schwartz_decoherence_inv_alpha": MATRIX(dyn_var.ndof, 1), "schwartz_interaction_width": MATRIX(dyn_var.ndof, 1),

```

Comment:
this would break the backward-compatibility, please make sure the old way also works, while introducing a more general way